pydicom/dataset.py

```diff
@@ -793,8 +743,10 @@ def convert_pixel_data(self):
         None
             Converted pixel data is stored internally in the dataset.
 
-        If a compressed image format, the image is  decompressed,
-        and any related data elements are changed accordingly.
+        Notes
+        -----
+        If the pixel data is in a compressed image format, the data is
+        decompressed and any related data elements are changed accordingly.
```

Comment:
Is that also correct for Photometric Interpretation? I remember some disussion that some handlers do not set this correctly after converting color space.

For something completely different (just noticed while writing "color space", and having read "convert_colour_space" above) - do we have some convention to use American or British English? Right now it seems to be starting to get a mix, which I personally find a bit confusing, especially if used in the public API.

Reply:
I'm in favour (hah) of supporting both, but generally try to avoid names where its an issue.  

As you noted below, I did add an Americanised version.